task/fbc-validation/0.1/fbc-validation.yaml

```
@@ -172,6 +177,25 @@ spec:
           failure_num=`expr $failure_num + 1`
           TESTPASSED=false
         fi
+        if [[ "${OCP_VER_MAJOR}" -ge "${OCP_BUNDLE_METADATA_THRESHOLD_MAJOR}" ]] && [[ "${OCP_VER_MINOR}" -ge "${OCP_BUNDLE_METADATA_THRESHOLD_MINOR}" ]]; then
+          #
+          # above the version threshold for mandatory `olm.csv.metadata` bundle metadata format, so presence of `olm.bundle.object` is an error
+          #
+          if ! ${OPM_BINARY} render ."${conffolder}" | jq -en 'reduce( inputs | select(.schema == "olm.bundle" and .properties[].type == "olm.bundle.object")) as $_ (0;.+1) == 0'; then
+            echo "!FAILURE! - olm.bundle.object is not permitted in a FBC fragment for OCP version ${OCP_VER_MAJOR}.${OCP_VER_MINOR}. Fragments must move to olm.csv.metadata bundle metadata."
+            failure_num=`expr $failure_num + 1`
+            TESTPASSED=false
+          fi
+        else
+          #
+          # below the version threshold for mandatory `olm.csv.metadata` bundle metadata format, so presence of `olm.csv.metadata` is an error
+          #
+          if ! ${OPM_BINARY} render ."${conffolder}" | jq -en 'reduce( inputs | select(.schema == "olm.bundle" and .properties[].type == "olm.csv.metadata")) as $_ (0;.+1) == 0'; then
+            echo "!FAILURE! - olm.csv.metadata is not permitted in a FBC fragment for OCP version ${OCP_VER_MAJOR}.${OCP_VER_MINOR}. Fragments must only use olm.bundle.object bundle metadata."
```

```suggestion
            echo "!FAILURE! - olm.csv.metadata bundle properties are not permitted in a FBC fragment for OCP version ${OCP_VER_MAJOR}.${OCP_VER_MINOR}. Fragments must only use olm.bundle.object bundle metadata."
```

And then similar suggestion as above for telling folks how to fix it?